gradoop-flink/src/test/java/org/gradoop/model/impl/GraphTransactionTest.java

```diff
@@ -63,4 +63,31 @@ public void testTransformationWithCustomReducer() throws Exception {
       originalCollection.equalsByGraphData(restoredCollection));
   }
 
+  @Test
+  public void regressionTest273() throws Exception {
```

Comment:
please rename the test to be more descriptive (the 273 can be referenced in the javadoc, if necessary)
maybe `testWithSubsetGraphContainment()`?